packages/common/src/FormattingUtils.js

```diff
@@ -124,9 +124,9 @@ function addSign(number) {
   }
 }
 
-// Take an amount based on fromDecimals and convert it to an amount based on toDecimals
```

Comment:
Updated the comment here to be clearer.

Reply:
Do we need to handle the case where `fromDecimals > toDecimals`?

Reply:
The util does this already :)